vscode-ext/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
 	"name": "mirrord",
```

Comment:
Note that you've left the version as 1.0.0 here

Reply:
Sorry, for wasting your time, I messed up such a simple issue.

Reply:
No worries at all! Took me like two seconds and we're glad to finally get this issue done :)